agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1735,10 +1735,32 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<SSlaveSocke
             return false;
         }
 
+        //remove this client from other radios
+        for (const auto &slave : m_radio_info_map) {
+            auto associated_clients_map = slave.second.associated_clients_map;
+            for (const auto &vap : associated_clients_map) {
+                if (vap.second.find(msg->client_mac()) != vap.second.end()) {
+                    m_radio_info_map.erase(msg->client_mac());
+                }
+            }
+        }
         // Set client association information for associated client
+        std::string assoc_req;
+        if (!msg->association_frame()) {
```

Comment:
Why not simply leave it as empty string? no need for the debug print and if else.
```suggestion
        std::string assoc_req(msg->association_frame() ? msg->association_frame() : "");
```